web_last_viewed_records/static/src/xml/main.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name="web_last_viewed_records.header">
+        <tr class="oe_header_row">
+            <td t-att-colspan="colspan or '3'">
+                <div class="oe_view_manager_last_viewed"/>
+            </td>
+            <td></td>
```

Comment:
Hi, 
Excellent feature :+1:  Thanks for the localstorage test. 
One thing :
This line (empty td) generate an empty space useless. Is it usefull ? 

I tried the following code and it works better. But I don't know the impact on mail.wall...

    t t-name="web_last_viewed_records.header"
        tr class="oe_header_row"
            td t-att-colspan="'4'"
                div class="oe_view_manager_last_viewed"/
            /td
        /tr
    /t
![capture du 2015-03-16 15_34_27](https://cloud.githubusercontent.com/assets/3407482/6668296/33b3bda2-cbf2-11e4-8b15-9dac05f179ad.png)